common/src/main/java/com/yahoo/omid/YAMLUtils.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid;
+
+import com.google.common.base.Joiner;
+import com.google.common.io.Resources;
+import org.apache.commons.beanutils.BeanUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+
+public class YAMLUtils<T> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(YAMLUtils.class);
+
+    private Joiner joiner = Joiner.on(", ");
+
+    public void loadDefaultSettings(List<String> fileNames, Object bean) {
+
```

Comment:
Looks like you manually added empty lines everywhere. Although I don't have a strong opinion about whitespace in particular, I'd prefer our style be coherent everywhere. If we want empty lines like that we need to change all other classes.